tui_examples/shell_using_salome_GUI_model/create_mdpa.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+
+
+try:

```

Comment:
I am not a fan of the `try` here but I know that `__file__` is somehow not working properly inside of salome
could you add a comment here explaining how to use this?

Reply:
I removed the `try` - since you integrated the exception handler, it is now not required anymore.

`__file__` is not working, since Salome loads the python script as string with `exec`.
Thats why two paths need to be manually added/updated:

- Path to plugin
- Path to output directory of .mdpa 

The latter however might be possible to retrieve from Salome since it should know the project directory...